SMS19/settings.py

```diff
@@ -97,7 +104,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [os.path.join(BASE_DIR, 'main','templates', 'allauth')],
```

Comment:
Add the templates directory for the Allauth templates .  We are not gonna show user the default simple templates of allauth .  And so this template dir. overrides the default templates